core/models.py

```diff
@@ -26,9 +26,16 @@ class Upload(models.Model):
     Upload object will store all information about received file and related processes.
     """
 
+    QUEUED_VALIDATION = "queued.validation"
+    VALIDATION = "validation"
+    STATUS_CHOICES = [
+        (QUEUED_VALIDATION, _("Queued validation")),
+        (VALIDATION, _("Validation")),
+    ]
     id = models.UUIDField(primary_key=True, editable=False, default=uuid.uuid4)
     filename = models.CharField(max_length=64)
```

Comment:
is it enough to hold whole path?

Reply:
in this field store only filename like `dataset.json`